README.md

```diff
@@ -218,20 +226,38 @@ To run the census functional tests within the cypress UI:
 
 ### Deployment with [Helm](https://helm.sh/)
 
-To deploy this application with helm, you must have a kubernetes cluster already running.
+To deploy this application with helm, you must have a kubernetes cluster already running and be logged into the cluster.
 
+Log in to the cluster using:
+```
+gcloud container clusters get-credentials survey-runner --region <region> --project <gcp_project_id>
+```
 You need to have Helm installed locally
 
 1. Install Helm with `brew install kubernetes-helm` and then run `helm init --client-only`
 
-1. Install Helm Tiller plugin for tillerless deploys `helm plugin install https://github.com/rimusz/helm-tiller`
+1. Install Helm Tiller plugin for *Tillerless* deploys `helm plugin install https://github.com/rimusz/helm-tiller`
 
-To deploy to a cluster you can run the following command
+Before deploying the app to a cluster you need to create the application credentials on Kubernetes. Run the following command to provision the credentials:
 
 ```
-helm tiller run helm upgrade --install survey-runner ./helm/runner --set projectId=PROJECT_ID --set submissionBucket=BUCKET_NAME 
+EQ_KEYS_FILE=PATH_TO_KEYS_FILE EQ_SECRETS_FILE=PATH_TO_SECRETS_FILE ./k8s/deploy_credentials.sh
+```
+##### Example
+```
+EQ_KEYS_FILE=docker-keys.yml EQ_SECRETS_FILE=docker-secrets.yml ./k8s/deploy_credentials.sh
```

Comment:
Any reason for this to use environment variables rather than arguments to the script (as `deploy_app.sh` does)?

Reply:
I've just realised that it's better as you have it, as otherwise someone could provide the paths to the files in the wrong order.